examples/embeddable_examples/public/react_embeddables/saved_book/types.ts

```diff
@@ -23,9 +22,10 @@ export interface BookAttributes {
   bookSynopsis?: string;
 }
 
-export type BookAttributesManager = {
-  [key in keyof Required<BookAttributes>]: BehaviorSubject<BookAttributes[key]>;
-} & { comparators: StateComparators<BookAttributes> };
+export type BookAttributesManager = StateManager<BookAttributes>;
```

Comment:
Remove `BookAttributesManager` and just use `StateManager<BookAttributes>` inline where needed. No need to create a type for something that is this simple